app/views/curation_concerns/base/_media_item.html.erb

```diff
@@ -0,0 +1,9 @@
+<tr class="track_listing"
+    data-title="<%= member.title.first %>"
+    data-member-id="<%= member.id %>"
+    data-mp3-url="<%= CHF::AudioDerivativeMaker.s3_url(file_set_id:member.id, file_checksum:member.representative_checksum, type_key: :standard_mp3) %>"
+    data-webm-url="<%= CHF::AudioDerivativeMaker.s3_url(file_set_id:member.id, file_checksum:member.representative_checksum, type_key: :standard_webm) %>"
+>
+    <!-- TODO instead of a table, make this a list of links.  -->
+    <td class="title" title="Listen to '<%= member.title.first %>' "><%= member_counter + 1 %>. <%= member.title.first %></td>
```

Comment:
merge this into _media_player.html.erb, don't think it needs to be a separate template, although I see why you did it it's not totally unreasonable.

Reply:
Yup, I folded these in.